test-infra/end-to-end-test/test_streaming_kafka010.sh

```diff
@@ -64,18 +65,18 @@ $KAFKA_DIR/bin/kafka-topics.sh --create --zookeeper localhost:2181 --replication
 $KAFKA_DIR/bin/kafka-topics.sh --create --zookeeper localhost:2181 --replication-factor 1 --partitions 1 --topic test-output
 
 # run the Flink job (detached mode)
-$FLINK_DIR/bin/flink run -d build-target/examples/streaming/Kafka010Example.jar \
+$FLINK_DIR/bin/flink run -d $FLINK_DIR/examples/streaming/Kafka010Example.jar \
```

Comment:
This ```Kafka010Example.jar``` is the use code and load this jar with child first classloading in this test ?

Reply:
Yes, by default child-first classloading is used.

Reply:
Thanks ~ Makes more sense to me now.